fili-core/src/main/java/com/yahoo/bard/webservice/web/apirequest/ApiRequestImpl.java

```diff
@@ -660,9 +660,8 @@ protected LogicalTable generateTable(
 
         // check if requested logical table grain pair exists
         if (generated == null) {
-            String msg = TABLE_GRANULARITY_MISMATCH.logFormat(granularity, tableName);
-            LOG.error(msg);
-            throw new BadApiRequestException(msg);
+            LOG.debug(TABLE_UNDEFINED.logFormat(tableName));
+            throw new BadApiRequestException(TABLE_UNDEFINED.format(tableName));
```

Comment:
`TABLE_GRANULARITY_MISMATCH ` is changed to `TABLE_UNDEFINED ` type of error message, because `logicalTableDictionary.get()` above has already identified the table-granularity-mismatch

Also, this `TABLE_GRANULARITY_MISMATCH ` is misleading, because having `generated == null` doesn't necessarily mean it's a mismatch; it could also be that table with this granularity does not exist in logicalTableDictionary at all.